src/modules/crud.js

```diff
@@ -0,0 +1,86 @@
+// Constants:
+const addTask = document.getElementById('input');
+const list = document.getElementById('tasks-list');
+
+function crud() {
+  list.innerHTML = '';
+  const tasksList = localStorage.getItem('data')
+    ? JSON.parse(localStorage.getItem('data'))
+    : [];
+  for (let i = 0; i < tasksList.length; i += 1) {
+    const input = document.createElement('input');
+    input.setAttribute('type', 'checkbox');
+    input.classList.add('checkbox');
+    input.id = i + 1;
+    if (tasksList[i].completed) {
+      input.checked = true;
+    }
+
+    const label = document.createElement('input');
+    label.value = `${tasksList[i].description}`;
+    label.classList.add('description');
+    label.setAttribute('type', 'text');
+    if (tasksList[i].completed) {
+      label.style.textDecorationLine = 'line-through';
+    }
+
+    label.setAttribute('readOnly', 'readOnly');
```

Comment:
- [ ] [OPTIONAL] This file contains a lot of codes and as we have learned how to split our codes into small individual files called modules I would suggest that you try to reduce the codes in this crud function by making small individual functions and then you place them in their individual files. Then in order to use them you know, you can use import/export javascript ES6 features 😉 The only way we master what we learn is by practicing them and you need to start with small projects like this 😎

Reply:
I will do this in later stages, but for now please consider the way I did. Thank you for the kind review.